myhpi/core/templates/core/minutes.html

```diff
@@ -6,82 +6,85 @@
-            <aside class="side-panel border-accent">
-              <h1 class="side-panel-title">{% translate "Location" %}</h1>
-              <p>{{ page.location }}</p>
-            </aside>
+<div class="row minutes-container">
+    {% with page.body|markdown as parsed_md %}
+    <h1 class="page-title">
+        <span class="underline">
+            {{ page.title }}
+        </span>
+    </h1>
+    <div class="col-9 minutes-text">
+        {{ parsed_md.0|touchify_abbreviations|tag_external_links }}
+    </div>
+    <div class="col-lg-3 minutes-meta">
+        <aside class="side-panel border-accent">
+            <h1 class="side-panel-title">{% translate "Date" %}</h1>
+            <p>{{ page.date }}</p>
+        </aside>
+        <aside class="side-panel border-accent">
+            <h1 class="side-panel-title">{% translate "Moderator" %}</h1>
+            <p>{{ page.moderator.get_full_name }}</p>
+        </aside>
+        <aside class="side-panel border-accent">
+            <h1 class="side-panel-title">{% translate "Minutes taker" %}</h1>
+            <p>{{ page.author.get_full_name }}</p>
+        </aside>
+        <aside class="side-panel border-accent">
+            <h1 class="side-panel-title">{% translate "Participants" %}</h1>
+            <ul>
+                {% for participant in page.participants.all %}
+                <li>{{ participant.get_full_name }}</li>
+                {% endfor %}
+            </ul>
+        </aside>
+        <aside class="side-panel border-accent">
+            <h1 class="side-panel-title">{% translate "Guests" %}</h1>
+            {% if page.guests %}
+            <ul>
+                {% for guest in page.guests %}
+                <li>{{ guest }}</li>
+                {% endfor %}
+            </ul>
+            {% else %}
+            <i>{% translate "No guests" %}</i>
+            {% endif %}
+        </aside>
+        {% if page.location %}
+        <aside class="side-panel border-accent">
+            <h1 class="side-panel-title">{% translate "Location" %}</h1>
+            <p>{{ page.location }}</p>
+        </aside>
+        {% endif %}
+        <aside class="side-panel border-accent">
+            <h1 class="side-panel-title">{% translate "Labels" %}</h1>
+            {% if page.labels.all %}
+            <p>{% include "core/label.html" with minutes=page %}</p>
+            {% else %}
+            <i>{% translate "No labels"%}</i>
+            {% endif %}
+        </aside>
+        {% include "core/sidebar.html" %}
+    </div>
+    <div id="minutes-footer" class="d-none d-print-block"></div>
+    <div id="minutes-navigation" class="row d-print-none">
+        <div class="col-4">
+            {% prev_minutes page as prev %}
+            {% if prev %}
+            <a href="{{ prev.url }}" class="btn btn-outline-primary minutes-navigation-button">{% bs_icon 'arrow-left'
+                extra_classes='minutes-navigation-icon' %} {% translate "Previous minutes" %}</a>
```

Comment:
Looks like this for me:
![image](https://github.com/fsr-de/myHPI/assets/6863832/86780420-b8ce-4789-97cc-02471df4a28b)

Reply:
Yeah I tweaked the design a bit /s

Reply:
Fixed